docs/UserGuide.md

```diff
@@ -91,7 +91,21 @@ Examples:
 
 Shows a list of all persons in the address book.
 
-Format: `list`
+Format: `list [s/FIELD]`
+
+* Sorts the contacts by the specified field in **ascending** order. `FIELD` must take one the following values:
+  * `d` Sort by date of birth
+  * `g` Sort by gender
+  * `n` Sort by number of products purchased
+  * `pa` Sort by purchase amount. i.e. Total amount ever spent
```

Comment:
I think we can remove this line since we will no longer be doing purchase amount.